tools/remote_build/windows.bazelrc

```diff
@@ -1,3 +1,49 @@
-# TODO(yfen): Merge with rbe_common.bazelrc and enable Windows RBE
```

Comment:
nit: still worth looking into what .bazelrc files we can include to avoid duplication. Fine to do as a followup.

Reply:
This file has to exist as an independent config for a while, because we don't want to change the default Linux RBE behavior. With the combined file, it requires Linux RBE to have additional arguments.